plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/editors/EditorContentFunction.java

```diff
@@ -81,13 +84,18 @@ public Object executeFunction(String functionName, Object[] parameters) {
 
         } else if (functionName.equals(HumantaskEditorConstants.JS_CUSTOMFUNC_REMOVE_WSDL)) {
             IWorkspace workspace = ResourcesPlugin.getWorkspace();
-            IResource resource = null;
-            IResource cbResource = null;
-            cbResource = workspace.getRoot().getProject(getProjectName())
-                    .findMember(parameters[1] + HumantaskEditorConstants.CALLBACK_TASK_WSDL_SUFFIX);
-            resource = workspace.getRoot().getProject(getProjectName())
-                    .findMember(parameters[1] + HumantaskEditorConstants.TASK_WSDL_SUFFIX);
-
+            IResource resource = workspace
+                    .getRoot()
+                    .getProject(getProjectName())
+                    .findMember(
+                            HumantaskEditorConstants.BASE_FOLDER_NAME + File.separator + parameters[1]
+                                    + HumantaskEditorConstants.TASK_WSDL_SUFFIX);
+            IResource cbResource = workspace
+                    .getRoot()
+                    .getProject(getProjectName())
+                    .findMember(
+                            HumantaskEditorConstants.BASE_FOLDER_NAME + File.separator + parameters[1]
```

Comment:
Please check all the places where you can substitute file system path using Paths.